packages/sql/README.md

```diff
@@ -7,23 +7,22 @@ A SQL toolkit for Effect.
 ```ts
 import { Config, Effect, Struct, pipe } from "effect"
 import * as Sql from "@effect/sql"
```

Comment:
You can remove this import now.

Reply:
woops - code in docs is annoying, I need my damn linter :D

don't want to bother you with reviews, but I think that's it...